Sources/SwifterSwift/SwiftStdlib/CollectionExtensions.swift

```diff
@@ -89,26 +81,47 @@ public extension Collection where Index == Int {
     ///   - slice: size of array in each interation.
     ///   - body: a closure that takes an array of slice size as a parameter.
     func forEach(slice: Int, body: ([Element]) throws -> Void) rethrows {
```

Comment:
I'd really love to make this `SubSequence` instead of `[Element]`, but I don't want a breaking change.

Reply:
That was discussed back when implemented a long time ago ... I agree, but not sure it's worth a breaking change at this point.